containers/attributes/main.py

```diff
@@ -976,7 +976,13 @@ async def delete_authorities_crud(request):
             table_authority.c.name == request.authority
         )
     )
-    await database.execute(statement)
+    try:
+        await database.execute(statement)
+    except ForeignKeyViolationError as e:
+        raise HTTPException(
+            status_code=status.HTTP_405_METHOD_NOT_ALLOWED,
+            detail=f"Delete failed due to foreign-key constraint: {str(e)}"
```

Comment:
Maybe rephrase to be more general instead of exposing the implementation error?
```suggestion
            detail=f"Unable to delete non-empty authority"
```

Reply:
Done